packages/miniflare/types/streams.d.ts

```diff
@@ -11,7 +11,7 @@ declare module "stream/web" {
 	}
 
 	interface ReadableStreamBYOBReader {
-		readonly closed: Promise<void>;
+		readonly closed: Promise<undefined>;
```

Comment:
updating typescript made this a little more strict (I think), so I had to fix this here